package.json

```diff
@@ -12,8 +12,8 @@
     "test": "lerna run test",
     "lint": "lerna run lint",
     "link": "lerna link",
-    "version": "lerna version --no-private --conventional-commits --create-release github --yes",
-    "publish": "lerna publish from-git --yes --no-verify-access",
+    "version": "lerna version --no-private --conventional-commits --create-release github --yes --exact",
+    "publish": "lerna publish from-git --yes --no-verify-access --exact",
```

Comment:
I might be wrong, but this doesn't seem to be a valid option? https://github.com/lerna/lerna/tree/main/commands/publish

Reply:
I actually think it is. In the docs above the options of `publish`:

 > lerna publish supports all of the options provided by lerna version in addition to the following:
 
 And `--exact` it's defined in `version`
 
 it took me also a while to see that line :D

Reply:
Ah good point - I went straight to the list of options